tests/endpoints/v1/test_beam.py

```diff
@@ -0,0 +1,787 @@
+    def test_get_analysis_group(self, client):
+        result = client.beam.analysis_group.get(group_id="abc123")
+
+        client.request.assert_called_once_with(
+            "get",
+            "/v1/beam/analysis-groups/abc123/",
+            params={},
+            verify=True,
+        )
+        assert isinstance(result, AnalysisGroup)
+
+    @with_mock_client()
+    def test_update_group_params_underscores(self, client):
+        client.beam.analysis_group.update(
+            group_id="abc123",
+            name="name",
+            analysis_ids=["analysis_id1", "analysis_id2"],
+            demand_type__unit_descriptor="unit_descriptor",
+        )
+
+        client.request.assert_called_once_with(
+            "patch",
+            "/v1/beam/analysis-groups/abc123/",
+            json={
+                "name": "name",
+                "analysis_ids": ["analysis_id1", "analysis_id2"],
+                "demand_type": {"unit_descriptor": "unit_descriptor"},
+            },
+            verify=True,
+        )
+
+    @with_mock_client()
+    def test_update_group_params_dicts(self, client):
+        client.beam.analysis_group.update(
+            group_id="abc123",
+            name="name",
+            analysis_ids=["analysis_id1", "analysis_id2"],
+            demand_type={"unit_descriptor": "unit_descriptor"},
+        )
+
+        client.request.assert_called_once_with(
+            "patch",
+            "/v1/beam/analysis-groups/abc123/",
+            json={
+                "name": "name",
+                "analysis_ids": ["analysis_id1", "analysis_id2"],
+                "demand_type": {"unit_descriptor": "unit_descriptor"},
+            },
+            verify=True,
+        )
+
+    @with_mock_client()
+    def test_delete_group(self, client):
+        client.beam.analysis_group.delete(group_id="abc123")
+
+        client.request.assert_called_once_with(
+            "delete",
+            "/v1/beam/analysis-groups/abc123/",
+            params={},
+            verify=True,
+        )
+
+    @with_mock_client()
+    def test_refresh_group(self, client):
+        client.beam.analysis_group.refresh(group_id="abc123")
+
+        client.request.assert_called_once_with(
+            "post",
+            "/v1/beam/analysis-groups/abc123/refresh/",
+            params={},
+            verify=True,
+        )
+
+    @with_mock_client(
+        request_returns=load_fixture(
+            "requests_responses/beam_test/test_empty_feature_importance"
+        )
+    )
+    def test_group_feature_empty_importance(self, client):
+        client.beam.analysis_group.get_feature_importance(group_id="abc123")
+
+        client.request.assert_called_once_with(
+            "get",
+            "/v1/beam/analysis-groups/abc123/feature-importance/",
+            params={},
+            verify=True,
+        )
+
+    @with_mock_client(
+        request_returns=load_fixture(
+            "requests_responses/beam_test/test_empty_feature_importance"
```

Comment:
is this the correct fixture for this test? I think yo might have swapped the names of the fixture files around for empty vs non-empty feature importance